crt_portal/cts_forms/views.py

```diff
@@ -436,31 +436,58 @@ def process_intake_filters(request):
     }
 
 
-def get_action_data(requested_actions):
+def get_action_data(requested_actions, report_url_args, paginated_offset):
     data = []
-    for action in requested_actions:
+    for index, action in enumerate(requested_actions):
         data.append({
             "action": action.verb,
             "detail": action.description,
             "timestamp": action.timestamp,
             "reportid": action.target_object_id,
-            "url": f'/form/view/{action.target_object_id}'
+            "url": f'/form/view/{action.target_object_id}?next={report_url_args}&index={paginated_offset + index}'
         })
     return data
 
 
 def process_activity_filters(request):
     query_filters, selected_actions = dashboard_filter(request.GET)
+    per_page = request.GET.get('per_page', 15)
+    page = request.GET.get('page', 1)
+    sort_expr, sorts = activity_sort(request.GET.getlist('sort'))
+    if selected_actions != []:
```

Comment:
is it necessary to compare explicitly to empty list here?

```suggestion
    if selected_actions:
```

Reply:
When it was not explicitly comparing to the empty list returned by the dashboard_filter function it was breaking because it was trying to perform an order_by method on a list.